docs/keys_and_certificates.md

```diff
@@ -0,0 +1,263 @@
+# Keys and certificates
+
+Many different keys and certificates are used by all of Fobnail's components.
+Most of them are created by the code, but some must be provided externally.
+
+## List of keys and certificates used
+
+### Platform Owner
+
+#### Knows in advance
+
+- Platform Owner key (signing and encryption)
+    - Fobnail and remote platform provisioning
```

Comment:
@krystian-hebel I don't know how to read that, what does this bullet mean?

Reply:
Maybe we should link to other sections in documentation to make sure terms are correctly understood?

Reply:
I'm not sure if table format in this case wouldn't be better, but with description it is way clearer.